services/registration/src/routes/index.ts

```diff
@@ -1,9 +1,56 @@
+import { asyncHandler } from "@api/common";
 import express from "express";
 
+import { ApplicationModel } from "src/models/application";
+import { BranchModel, BranchType } from "src/models/branch";
 import { applicationRouter } from "./application";
 import { branchRouter } from "./branch";
 
 export const defaultRouter = express.Router();
 
 defaultRouter.use("/branches", branchRouter);
 defaultRouter.use("/applications", applicationRouter);
+
+defaultRouter.route("/statistics").get(
+  asyncHandler(async (req, res) => {
```

Comment:
The `index.ts` file should be very barebones. As such, i would recommend creating a new file called statistics and then create a new router. Then, the `index.ts` file can just have a reference to that using like `defaultRouter.use("/statistics", statisticsRouter);`